src/Main.ps1

```diff
@@ -638,6 +638,11 @@ function Invoke-Pester {
         # this will inherit to child scopes and allow Describe / Context to run directly from a file or command line
         $invokedViaInvokePester = $true
 
+        if ($null -ne $state) {
+            # this will inherit to child scopes and affect behavior of ex. TestDrive/TestRegistry
+            $runningPesterInPester = $true
```

Comment:
Should we track this (and `$invokedViaInvokePester`) in `$state`-prop instead of inheriting the variables?

Reply:
Sure, why not.

Reply:
Saving for future PR :)